clients/java/dkv-client/src/main/java/org/dkv/client/ShardedDKVClient.java

```diff
@@ -260,6 +260,12 @@ public void delete(byte[] key) {
         return dkvClient.iterate(startKey, keyPref);
     }
 
+    @Override
+    public long getDbSize() {
+        //TODO: implement this behaviour
```

Comment:
Probably this shouldn't even be exposed to java client unless its an exact value.